WebHDFS.Kitty.sln

```diff
@@ -1,31 +1,37 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.645
-MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebHDFS.Kitty", "WebHDFS.Kitty\WebHDFS.Kitty.csproj", "{90E44E8F-254A-48BA-9387-2F359F6388AE}"
-EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebHDFS.Kitty.IntegrationTests", "WebHDFS.Kitty.IntegrationTests\WebHDFS.Kitty.IntegrationTests.csproj", "{ED88D91D-2F8C-40FE-B552-6558E4655E5F}"
-EndProject
-Global
-	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|Any CPU = Debug|Any CPU
-		Release|Any CPU = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{90E44E8F-254A-48BA-9387-2F359F6388AE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{90E44E8F-254A-48BA-9387-2F359F6388AE}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{90E44E8F-254A-48BA-9387-2F359F6388AE}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{90E44E8F-254A-48BA-9387-2F359F6388AE}.Release|Any CPU.Build.0 = Release|Any CPU
-		{ED88D91D-2F8C-40FE-B552-6558E4655E5F}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{ED88D91D-2F8C-40FE-B552-6558E4655E5F}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{ED88D91D-2F8C-40FE-B552-6558E4655E5F}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{ED88D91D-2F8C-40FE-B552-6558E4655E5F}.Release|Any CPU.Build.0 = Release|Any CPU
-	EndGlobalSection
-	GlobalSection(SolutionProperties) = preSolution
-		HideSolutionNode = FALSE
-	EndGlobalSection
-	GlobalSection(ExtensibilityGlobals) = postSolution
-		SolutionGuid = {E737FE30-CBCF-4B68-AF50-CC9792BF0C7B}
-	EndGlobalSection
-EndGlobal
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29509.3
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebHDFS.Kitty", "WebHDFS.Kitty\WebHDFS.Kitty.csproj", "{90E44E8F-254A-48BA-9387-2F359F6388AE}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebHDFS.Kitty.IntegrationTests", "WebHDFS.Kitty.IntegrationTests\WebHDFS.Kitty.IntegrationTests.csproj", "{ED88D91D-2F8C-40FE-B552-6558E4655E5F}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebHDFS.Kitty.XUnitTests", "WebHDFS.Kitty.XUnitTests\WebHDFS.Kitty.XUnitTests.csproj", "{14D67D2E-D412-4515-B130-180BCE987178}"

```

Comment:
`WebHDFS.Kitty.IntegrationTests` project should be removed, `WebHDFS.Kitty.XUnitTests` project and folder should be renamed to `WebHDFS.Kitty.IntegrationTests`